From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Gautham Srinivasan <gauthams@nvidia.com>
Date: Wed, 17 Jul 2024 10:22:32 +0000
Subject: [PATCH 240/243] NVIDIA: SAUCE: gpio: tegra186: Enable GTE for
 Tegra264

Set has_gte flag to enable GTE for Tegra264 AON pins.

Change-Id: I29c644e076c05d8b2ba3f33091599c19474e9eb5
Signed-off-by: Gautham Srinivasan <gauthams@nvidia.com>

diff --git a/drivers/gpio/gpio-tegra186.c b/drivers/gpio/gpio-tegra186.c
index 8dc2b964819d..87df62fb57b8 100644
--- a/drivers/gpio/gpio-tegra186.c
+++ b/drivers/gpio/gpio-tegra186.c
@@ -1466,6 +1466,7 @@ static const struct tegra_gpio_soc tegra264_aon_soc = {
 	.num_irqs_per_bank = 8,
 	.has_gte = true,
 	.has_vm_support = true,
+	.has_gte = true,
 };
 
 #define TEGRA264_UPHY_GPIO_PORT(_name, _bank, _port, _pins) \
-- 
2.43.0

